scripts/natural_language_inference/utils.py

```diff
@@ -0,0 +1,56 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# Copyright 2018 Mengxiao Lin <linmx0130@gmail.com>.
+
+"""
+Utility functions.
+"""
+
+import logging
+
```

Comment:
This also appeared in scripts/machine_translation/utils.py. It's not urgent but we should probably reuse code by moving this to gluonnlp.utils instead of duplication.